### ISSUE TITLE
.transition(transform 0.7s .3s ease) > repeated properites

### ISSUE THREAD

**Issue by Leagnus** [madebysource/lesshat#144]
.transition(transform 0.7s .3s ease);
makes
...
transition: -webkit-transform 0.7s 0.3s ease,-moz-transform 0.7s 0.3s ease,-o-transform 0.7s 0.3s ease,transform 0.7s 0.3s ease;

**Comment by planttheidea** [comment]
This is correct, because the transform property itself is prefixed. You need to replicate all prefixes to apply to all respective transforms.

**Comment by Leagnus** [comment]
no, i mean 2 times repeat
-webkit-transition: -webkit-transform 0.7s 0.3s ease;
		-moz-transition: -moz-transform 0.7s 0.3s ease;
		-o-transition: -o-transform 0.7s 0.3s ease;
		transition: -webkit-transform 0.7s 0.3s ease,-moz-transform 0.7s 0.3s ease,-o-transform 0.7s 0.3s ease,transform 0.7s 0.3s ease;
